modes/longitudinal/src/index.js

```diff
@@ -161,8 +161,15 @@ function modeFactory() {
             id: ohif.layout,
             props: {
               leftPanels: [tracked.thumbnailList],
+              // leftPanelDefaultState: {
+              //   closed: true,
+              //   openWhenPanelActivated: true,
+              // },
               rightPanels: [dicomSeg.panel, tracked.measurements],
-              // rightPanelDefaultClosed: true, // optional prop to start with collapse panels
+              rightPanelDefaultState: {
+                closed: true,
+                openWhenPanelActivated: true,
```

Comment:
I guess we can just use the old one which was `rightPanelDefaultClosed` and we don't need the `openWhenPanelActivated` anymore, in the SidePanel you can just check if that panel has some registered triggers, if so it means `openWhenPanelActivated`, right?

Reply:
There is no way to check if a panel has triggers registered because the PanelService does NOT store the triggers. So I think we should leave it like this.

Reply:
Furthermore, because the `PanelService` has a `activatePanel` method anyone can call that to activate the panel WITHOUT a trigger. So I am definitely going to leave it as is.